documentation/src/main/doc/modules/kafka/pages/avro-configuration.adoc

```diff
@@ -0,0 +1,77 @@
+[#kafka-avro-configuration]
+== Using Apache Avro serializer/deserializer
+
+If you are using https://avro.apache.org/[Apache Avro] serializer/deserializer, please note the following properties:
+
+=== For https://docs.confluent.io/current/schema-registry/serdes-develop/serdes-avro.html[Confluent] Schema Registry
+
+==== Consumer
+
+[options="header"]
+|===
+| "value.deserializer" | "io.confluent.kafka.serializers.KafkaAvroDeserializer"
+| "schema.registry.url" | "http://<your_host>:<your_port>/api/ccompat"
```

Comment:
you need CC Compat even on the confluent registry?

Reply:
O, Yes this is when you use Apicurio but confluent compatible. Let me fix that.